stories/Atom/Navigation/Sidebar/sidebar.scss

```diff
@@ -0,0 +1,42 @@
+@import "../../../assets/scss/mixins";
+@import "../../../assets/scss/breakpoints";
+@import "../../../assets/scss/variables";
+@import "../../../assets/scss/typography";
+
+// * Slidebar start *//
+.Sidebar_item {
+  @extend %border-bottom;
+  position: relative;
+  padding: 16px;
+  cursor: pointer;
+  overflow: hidden;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  @include devicebreak(small) {
+    padding: 24px;
+  }
+  &:hover {
+    background-color: $color-gray-200;
+  }
+  &.open {
+    .arrow {
+      transform: rotateZ(180deg);
+      transition: all 0.1s ease;
+    }
+  }
+  h6 {
+    margin-bottom: 0;
+  }
+  &--Selected {
+    background-color: $color-blue-600;
+    h6 {
+      color: $color-white;
+    }
+    &:hover {
+      background-color: $color-blue-600;
+    }
+  }
+}
+
+// * Slidebar end *//
```

Comment:
@harishkumar-srijan sidebarwitharrow has similar styles...please create the mixins if we have similar block of stylings

Reply:
@surbhi90  its done, placeholer created here.